nilearn/plotting/displays/_slicers.py

```diff
@@ -439,20 +439,20 @@ def _map_show(
         # is called from `add_contours`, continuous interpolation
         # does not make sense and we turn to nearest interpolation instead.
 
+        if is_binary_niimg(img):
+            resampling_interpolation = "nearest"
+
+        img = reorder_img(
+            img, resample=resampling_interpolation, copy_header=True
+        )
+
```

Comment:
I think this code might go inside `sanitize_transparency` or I can put a comment to indicate that the order of calling `reorder(img, ..)` before `sanitize_transparency` is important.